README.md

```diff
@@ -79,6 +79,52 @@ To launch Calico, install the default custom resource:
 To tear down the cluster:
 
 	make cluster-destroy
+	
+#### Running a custom image in your existing Calico (Enterprise) cluster
+
+These steps assume that you already have installed the operator in a Calico (Enterprise) cluster after following either 
+docs.projectcalico.org or docs.tigera.io. To verify, run `kubectl get deployment -n tigera-operator tigera-operator`. 
+You should see an existing deployment. 
+
+These are the steps:
+1. Make your own code changes to this repository.
+2. Create the binaries and a docker image.
+   ```bash
+   make build
```

Comment:
`make image` should produce the binaries for you, so no need to run this as two steps technically.